src/components/Tool/FixMailingAddresses/FixMailingAddresses.tsx

```diff
@@ -94,12 +99,9 @@ const useStyles = makeStyles()(() => ({
     },
   },
 }));
-
-const appName = process.env.APP_NAME || 'MPDX';
-
 export const emptyAddress: ContactAddressFragment = {
   id: 'new',
-  source: appName,
+  source: '',
```

Comment:
Does this mean we're now not saving the source when a new address is made? Or does it default to the appName on the server?

Reply:
@caleballdrin "MPDX" is the default value for addresses, email_addresses, contacts, and phone_numbers. So if you're not sending one, it'll default to that.

Reply:
Ok that is good to know. Thanks!